casbin/internal_enforcer.py

```diff
@@ -37,6 +54,23 @@ def _remove_policy(self, sec, ptype, rule):
 
         return rule_removed
 
+    def _remove_policies(self, sec, ptype, rules):
+        """removes rules from the current policy."""
+        for rule in rules:
+            rule_removed = self.model.remove_policy(sec, ptype, rule)
+            if not rule_removed:
+                continue
+
+            if self.adapter and self.auto_save:
+                if self.adapter.remove_policy(sec, ptype, rule) is False:
+                    continue
+
+                if self.watcher:
+                    self.watcher.update()
+
+        return True
+
```

Comment:
refer to: https://github.com/casbin/casbin/blob/master/internal_api.go#L101

Reply:
I'll have a try.